src/actions/guides/database/models.cr

```diff
@@ -7,70 +8,160 @@ class Guides::Database::Models < GuideAction
 
   def markdown
     <<-MD
-    ## Setting up a model
+    ## Introduction
+
+    A Model is an object used to map a corresponding database table to a class. These objects model real-world objects to give you a better understanding on how they should interact within your application.
+
+    Models in Lucky allow you to define methods associated with each column in the table. These methods return the value set in that column.
+
+    ## Generate a model
 
-    Let's create a `User` model with `lucky gen.model User` and then add some
-    columns. The name is a `String` and the age is an `Int32?`. The `age` column
-    allows null because the type is nilable (designated by the `?`).
+    Lucky gives you a task for generating a model along with several other files that you will need for interacting with your database.
+
+    Use the `lucky gen.model {ModelName}` task to generate your model. If you're generating a `User` model, you would run `lucky gen.model User`. Running this will generate a few files for you.
+
+    * [User model](##{ANCHOR_SETTING_UP_A_MODEL}) - Located in `./src/models/user.cr`
+    * [User form](#{Guides::Database::ValidatingSavingDeleting.path}) - Located in `./src/forms/user_form.cr`
+    * [User query](#{Guides::Database::Querying.path}) - Located in `./src/queries/user_query.cr`
+    * [User migration](#{Guides::Database::ManagingAndMigrating.path}) - Location in `./db/migrations/#{Time.utc.to_s("%Y%m%d%H%I%S")}_create_users.cr`
+
+    #{permalink(ANCHOR_SETTING_UP_A_MODEL)}
+    ## Setting up a model
 
-    > Note: we'll assume we ran a migration that created a name and age column.
+    Once you run the model generator, you'll have a file that looks like this
 
     ```crystal
     # src/models/user.cr
     class User < BaseModel
       table :users do
-        column name : String
-        column age : Int32?
+        # You will define columns here
```

Comment:
I kind of like having the columns, but agree it is odd to just have them there randomly. Maybe `# You will define columns here. For example:` and then a couple columns. What do you think?